### DISCUSSION TITLE
The plugin failed to initialize. Are you using FlutterFragmentActivity

### DISCUSSION THREAD

**Discussion by vytautas-pranskunas-** [flutter-stripe/flutter_stripe#464] [Q&A]
@jonasbark you promised to reopen closed ticket if i provide sample project so i did but ticket remains closed. SO I am opening new one - would you mind to look at it? 

https://github.com/flutter-stripe/flutter_stripe/issues/411

Thanks.

**Comment by jonasbark** [comment]
I'm only going to repeat myself:
> Please don't open a new ticket every time. I am unwilling to help any further if this behavior continues.

So I'm also closing this one. 
Closely compare your app with the example app by doing a side-by-side diff.

**Comment by jonasbark** [comment]
Actually you are right - this shouldn't be closed but be moved to Q&A discussion as it's a configuration issue 👍

**Reply by deathcoder** [reply]
We have the same problem, any hint on what the problem might be?

**Reply by jonasbark** [reply]
Please check the readme and follow it closely

**Reply by deathcoder** [reply]
I did to the best of my understanding, if you can help me figure out what I may have done wrong the changes are in these 2 commits:
 - https://github.com/striveapp/checkin/commit/5421874a73745933747de90edd7ec7c45a09be71
 - https://github.com/striveapp/checkin/commit/f0f3859fd47ecbebd95c3a75e27461ffda655e31

in the second one, I noticed a problem in the style files and I tried to remove and reinstall the Gradle wrapper to make sure it's using the correct version

**Reply by jonasbark** [reply]
According to https://github.com/striveapp/checkin/blob/master/android/app/src/main/AndroidManifest.xml#L15 you're still not using FlutterFragmentActivity for your MainActivity
and according to https://github.com/striveapp/checkin/blob/master/android/app/src/main/AndroidManifest.xml#L19 you're not using the NormalTheme that you added

**Reply by deathcoder** [reply]
You are right, I was able to fix the error with these changes, I also had to to a few more fixes because for my old project I wasn't compiling kotlin

**Comment by vytautas-pranskunas-** [comment]
@jonasbark so in order to get rid of this error I have to change:
from: android:name=".MainActivity"
to: android:name="io.flutter.embedding.android.FlutterActivity"
 
if it is so rather than closing multiple tickets it would so musch easier and faster to just move this issue to discussion and add this answer for everybody... 

I just do not remember where this .MainActivity is used.. somewhere I am using it either in Firebase profile either in Facebook login or maybe pushes configuration. So question is - how it will effect this?

**Reply by jonasbark** [reply]
If your MainActivity class already extends from FlutterFragmentActivity then there's nothing for you to do.

**Reply by vytautas-pranskunas-** [reply]
My .MainActivity file has this:

```
package com.example.test_app

import io.flutter.embedding.android.FlutterActivity
import io.flutter.embedding.android.FlutterFragmentActivity

class MainActivity: FlutterFragmentActivity() {
}

```

so I assume it does.. But... this does not remove an error.

**Reply by deathcoder** [reply]
@vytautas-pranskunas- I'm not sure we had the same problem, my fix was to make the change in the opposite way that you described, so I did:

from: android:name="io.flutter.embedding.android.FlutterActivity"
to: android:name=".MainActivity" 

but I also had to do a few more changes after that, if you want to take a look at the commit these are the things I changed to make it work:

https://github.com/striveapp/checkin/commit/a5ce36f4842c4d9255b9ff47f49b1a3dda2d434a
